.github/workflows/ci.yml

```diff
@@ -5,26 +5,21 @@ on:
   pull_request:
     branches: [ master ]
 
-jobs:
-  build:
+env:
+  AREA: liechtenstein
 
-    name: Compile, install and build mbtiles
+jobs:
+  generate-tiles:
     runs-on: ubuntu-latest
-
     steps:
-    - uses: actions/checkout@v2
-
-    - name: Install dependencies
-      run: |
-        sudo apt-get update
-        sudo apt-get install -y --no-install-recommends build-essential liblua5.1-0 liblua5.1-0-dev libprotobuf-dev libsqlite3-dev protobuf-compiler shapelib libshp-dev libboost-program-options-dev libboost-filesystem-dev libboost-system-dev
+    - name: Check out repository
+      uses: actions/checkout@v2
 
-    - name: Build and install
-      run: |
-        make -j 2
-        sudo make install
+    - name: Download PBF file
+      run: curl http://download.geofabrik.de/europe/${AREA}-latest.osm.pbf -o ${AREA}.osm.pbf
 
-    - name: Build openmaptiles-compatible mbtiles files of Liechtenstein
-      run: |
-        curl http://download.geofabrik.de/europe/liechtenstein-latest.osm.pbf -o liechtenstein.osm.pbf
-        tilemaker liechtenstein.osm.pbf --config=resources/config-openmaptiles.json --process=resources/process-openmaptiles.lua --output=liechtenstein.mbtiles --verbose
+    - name: Build openmaptiles-compatible mbtiles files of given area
+      uses: ./
+      with:
+        input: ${{ env.AREA }}.osm.pbf 
```

Comment:
I would not have a problem with these files always having the same name but if you want get fancy you can make them overridable.

Reply:
Same. Take a look in `action.yml` and `README.md`